src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -418,6 +430,13 @@ public override AbstractFormattingRule WithOptions(AnalyzerConfigOptions options
                 return CreateAdjustSpacesOperation(1, AdjustSpacesOption.ForceSpacesIfOnSingleLine);
             }
 
+            // Slice pattern:
+            // ..var x
```

Comment:
i keep flip flopping.  but i think i do prefer this formatting. should we check with mads/fred/ldm to see if anyone thinks otherwise?

Reply:
This PR is still in draft and I used an arbitrary (non-final) choice. It'll be easy to adjust to 0 or 1 space.
We'll bring this up at LDM on Monday with other list-pattern questions if there is time left.